screen-capture/getdisplaymedia.https.html

```diff
@@ -81,8 +78,7 @@
  {video: {frameRate: {exact: 4}}},
  {video: false, audio: true},
 ].forEach(constraints => promise_test(async t => {
-  await test_driver.bless('getDisplayMedia()');
-  const p = navigator.mediaDevices.getDisplayMedia(constraints);
+  const p = getDisplayMedia(constraints);
```

Comment:
This `p` is used below to assert that the returned promise was already rejected, but that doesn't really work if there's another `await` before it's raced with a `Promise.resolve()`. I'd suggest dropping the `getDisplayMedia` wrapper here and doing the timing check right after the call to `navigator.mediaDevices.getDisplayMedia(constraints)`.